### DISCUSSION TITLE
Progetto Esame Vittorio Dedi

### DISCUSSION THREAD

**Discussion by vittoorioo** [deneb-alpha/Lab_open_source-2023_2024#21] [Projects]
Parte Principale:
       Modelli di Business nel mondo Open Source 
       Localizzazione Libre Office con il Team italiano di traduzione
Parte Facoltativa: 
       Esercizi pratici sulla riproduzione dei bug con Bugzilla

**Comment by deneb-alpha** [comment]
Ciao @vittoriodedi,
I modelli di business open source sono sicuramente un valido argomento da presentare. A lezione abbiamo visto una carrellata di possibili modelli, all'esame, potrebbe essere interessante anche aggiungere qualche dettaglio in più con esempi di progetti che utilizzano quei modelli. 
Ovviamente questo è solo un suggerimento :) 
Per la localizzazione, ricordati di documentare anche come ti sei avvicinato al gruppo e come hai interagito con loro.

Per la parte opzionale su bugzilla, puoi anche prepararti dei bug triage prima dell'esame e durante la prova discutere i passi che vorresti fare per il triage etc. ti suggerirei di evitare comunque di lavorare su bugreport che non hai visto prima perché, come abbiamo visto anche a lezione, non va proprio tutto liscio.
